content/docs/next-release/frontend-ui.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Frontend/UI
+weight: 10
+---
+
+Jaeger Web UI is implemented in Javascript using popular open source frameworks like React. Several performance improvements have been released in v1.0 to allow the UI to efficiently deal with large volumes of data, and to display traces with tens of thousands of spans (e.g. we tried a trace with 80,000 spans).
+
+## Embed Mode
+
+Enhancement to embed mode in Jaeger UI for some components allowing us to integrate it into other applications.
+For use this feature we need to pass in the query params the param **uiEmbed** with the value **vN**, where **N** represents the version.
+
+
+### Embedded Components
+
+* Search Trace Page
+* Trace Page
+
+#### Search Trace Page
+
+To integrate the Search Trace Page to our application we have to indicate to the Jaeger UI that we want to use the embed mode with `uiEmbed=v0`. 
+
+For example:
+
+```sh
+<JAEGER UI URL>/search?
+    end=1543921359557000&
+    limit=20&lookback=1h&
+    maxDuration&
+    minDuration&
+    service=jaeger-query&
+    start=1543917759557000&
+    uiEmbed=v0
+```
+
+![Embed Search Traces](/img/frontend-ui/embed-search-traces.png)
```

Comment:
The image URLs need `/static/` prepended.

The images will also need to be updated whenever we land on a final layout and design.